src/pages/[locale]/sketches/[...slug].astro

```diff
@@ -1,39 +1,25 @@
 ---
+import SketchLayout from "@/src/layouts/SketchLayout.astro";
 import { nonDefaultSupportedLocales } from "@i18n/const";
-import BaseLayout from "@layouts/BaseLayout.astro";
-import { getCollectionInLocaleWithFallbacks } from "@pages/_utils";
-import { removeLocalePrefix } from "@i18n/utils";
+import { getCurationSketches } from "@/src/api/OpenProcessing";
 
 export async function getStaticPaths() {
+  // this will be the same for all locales since these aren't localized
+  // TODO: remove limit once curation source is real
+  const sketches = await getCurationSketches(25);
```

Comment:
Do we want to make that number 25 a const or the default value of the param? Just to make it easier for future editing to be sure that it is thorough?